operating/manage-compliance/managing-compliance-10.adoc

```diff
@@ -1,8 +1,8 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="manage-compliance"]
-= Managing compliance
+[id="managing-compliance-10"]
+= Managing compliance (1.0)
```

Comment:
@kcarmichael08 ^^

Reply:
I think if we change the menus to remove parentheses it would be an OK compromise? I mean, we are technically "managing compliance with the Compliance 2.0 feature", but that's even wordier....